front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -26,10 +38,20 @@ const StdcmViewV2 = () => {
         <StdcmConfig
           currentSimulationInputs={currentSimulationInputs}
           pathProperties={pathProperties}
+          isPending={isPending}
+          launchStdcmRequest={launchStdcmRequest}
+          cancelStdcmRequest={cancelStdcmRequest}
           setPathProperties={setPathProperties}
           setCurrentSimulationInputs={setCurrentSimulationInputs}
         />
       )}
+      {trainScheduleV2Activated && stdcmV2Results?.stdcmResponse && (
+        <StdcmResults
+          // TODO: Next step : use currentSimulationInputs instead of stdcmSimulationResults to handle multiples simulations results
+          stdcmData={stdcmResponse as StdcmV2SuccessResponse}
```

Comment:
Remove the const stdcmResponse and directly give it like this
```suggestion
          stdcmData={stdcmV2Results?.stdcmResponse}
```

Reply:
fix